frontend/static/js/exceptions/NotImplementedException.js

```diff
@@ -1,9 +1,9 @@
 
-class NotImplentedError extends Error {
+class NotImplementedError extends Error {
 	constructor(message) {
 		super(message);
 		this.name = "NotImplementedError";
```

Comment:
nice

Reply:
I think the typo is actually from me from a previous pr xD